tools/somatic_sniper/somatic_sniper.xml

```diff
@@ -0,0 +1,127 @@
+<tool id="somatic_sniper" name="Somatic Sniper" version="1.0.4">
+    <description> identify single nucleotide positions that are different between tumor and normal</description>
+<!--     <requirements>
+        <container type="docker">mutect:1.1.5</container>
```

Comment:
You can uncomment this section and change the name of the container to somatic-sniper:1.1.0.